app/src/components/modal/modal_lock_yo_tokens/index.tsx

```diff
@@ -61,15 +52,6 @@ const ModalMain = styled.div`
     margin-bottom: 12px;
   }
 `
-const LightDataItem = styled.div`
-  color: ${props => props.theme.textfield.placeholderColor};
-`
-const DarkDataItem = styled.div`
-  display: flex;
-  align-items: center;
-  color: ${props => props.theme.colors.textColorDark};
-  font-weight: ${props => props.theme.textfield.fontWeight};
-`
```

Comment:
@Frankaus So we should only use pure inline TYPE.something when there is one instance of it. Where we have repeating styles like this we should still use styled classes but instead of div you can put type like this.
const LightDataItem = styled(TYPE.bodyMedium) and try to incorporate color into styled. 
Because you are now repeating styles inline which is not good at all...